assemble/build.sh

```diff
@@ -138,8 +138,8 @@ Please vote one of:
 [ ] -1 - Because..., I do not accept...
 ... these artifacts as the $(green "$ver") release of Apache Accumulo.
 
-This vote will end on $(green "$votedate")
-($(green "$edtvotedate") / $(green "$pdtvotedate"))
+This vote will remain open until at least $(green "$votedate") ($(green "$edtvotedate") / $(green "$pdtvotedate")).
```

Comment:
I'd prefer to keep the newline between the UTC date on the first line and the convenient timezone-specific dates on the next, because the line wrapping can make these hard to read otherwise.